packages/core/review-workflows/admin/src/routes/settings/components/WorkflowAttributes.tsx

```diff
@@ -196,5 +203,57 @@ const ContentTypeTakeNotice = styled(Typography)`
   font-style: italic;
 `;
 
+/* -------------------------------------------------------------------------------------------------
+ * StageSelector
+ * -----------------------------------------------------------------------------------------------*/
+interface StageSelectorProps {
+  disabled?: boolean;
+}
+
+const StageSelector = ({ disabled }: StageSelectorProps) => {
+  const { value: stages = [] } = useField<WorkflowStage[]>('stages');
+  const { formatMessage } = useIntl();
+
+  const { error, value, onChange } = useField('stageRequiredForPublish');
+
+  return (
+    <Field.Root
+      error={error}
+      name={'stageRequiredForPublish'}
+      hint={formatMessage({
+        id: 'settings.review-workflows.workflow.stageRequiredForPublish.hint',
+        defaultMessage: 'Prevents entries to be published if they are not at the required stage.',
```

Comment:
I think it should be "from being" instead of "to be"? Could a native speaker check? @hanpaine @cache-your-dreams